etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/messagelink/MessageLinkTest.groovy

```diff
@@ -0,0 +1,117 @@
+package gov.hhs.cdc.trustedintermediary.etor.messagelink
+
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import spock.lang.Specification
+
+class MessageLinkTest extends Specification {
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+    }
```

Comment:
There's no `@Inject` or calls to the `ApplicationContext` in `MessageLink`.  You can remove this set-up.